### ISSUE TITLE
Buttons not working

### ISSUE THREAD

**Issue by Ayush1325** [phuhl/linux_notification_center#49]
So, I am not able to use any of the buttons in the notification center. The buttons show up just fine and hover colour and all work, but when I click them, the notification center closes and well, nothing happens. And this is true for all buttons, even the "delete all" button is not working.

I am using arch-linux with qtile as my window manager. I have also installed notify-send.py. So any help?

**Comment by phuhl** [comment]
Hey there, thanks for reporting and sorry for the late answer.

Does the notification center print any error messages when clicking a button?

**Comment by Ayush1325** [comment]
> Hey there, thanks for reporting and sorry for the late answer.
> 
> Does the notification center print any error messages when clicking a button?

Nope, it's almost like qtile doesn't even detect it. So even though the side panel shows, I can click on the window under the notification panel. The buttons on the notifications themselves do work though

**Comment by phuhl** [comment]
This seems to be a window-manager specific issue, then
